spec/system/topic_open_draft_button_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+RSpec.describe "Topic footer shows 'Open Draft' when draft exists", type: :system do
+  fab!(:post)
+  let(:topic) { post.topic }
+  let!(:second_post) { Fabricate(:post, topic: topic) }
+  fab!(:current_user) { Fabricate(:user) }
+
+  let(:topic_page) { PageObjects::Pages::Topic.new }
+  let(:composer) { PageObjects::Components::Composer.new }
+  let(:toasts) { PageObjects::Components::Toasts.new }
+
+  before { sign_in(current_user) }
+
+  it "changes the reply button label and tooltip for footer and post reply buttons, persists on reload" do
+    # Visit topic and verify initial footer label and tooltip are for Reply
+    topic_page.visit_topic(topic)
+    footer_button = find("#topic-footer-buttons .create")
+    expect(footer_button[:title]).to eq(I18n.t("js.topic.reply.help"))
+    expect(footer_button).to have_text(I18n.t("js.topic.reply.title"))
+
+    # Verify initial post-level reply button tooltip/label
+    post_button = find(".post-action-menu__reply", match: :first)
+    expect(post_button[:title]).to eq(I18n.t("js.post.controls.reply"))
+    expect(post_button).to have_text(I18n.t("js.topic.reply.title"))
```

Comment:
All this should move to page object, we already have `selector_for_post_action_button`, AI is great at these refactors